### PR TITLE
enhancenment to ceph-qe-scripts to run rhcs 5.0

### PR DESCRIPTION
**Pull request by mkasturi18**
Add support to run tests for RHCS 5.0 in ceph-qe-scripts.
executed the following tests against ceph releases luminous, nautilus and pacific.

tests:
test_Mbuckets_with_Nobjects.py -c configs/test_Mbuckets_with_Nobjects.yaml
test_Mbuckets_with_Nobjects.py -c configs/test_Mbuckets_with_Nobjects_delete.yaml

logs:
http://magna002.ceph.redhat.com/ceph-qe-logs/madhavi/PRs/cephadm/release/

### CODE REVIEW

File: rgw/v2/utils/utils.py [hunk 1]
```diff
@@ -170,7 +194,7 @@ def get_all_in_dir(path):
 
 def gen_bucket_name_from_userid(user_id, rand_no=0):
     log.info('generating bucket name or basedir to create')
-    bucket_name_to_create = user_id + "_" + BUCKET_NAME_PREFIX + "_" + str(rand_no)
+    bucket_name_to_create = user_id + "-" + BUCKET_NAME_PREFIX + "-" + str(rand_no)
```
Comment (rakeshgm) [inline rgw/v2/utils/utils.py:197]: using `_`  worked in nautilus. If this is not working in rhcs 5.0. we need to fie a BZ. can you add the bz here as a comment once you file it.

Comment (rakeshgm) [reply rgw/v2/utils/utils.py:197]: will wait for the bz before we merge.

**Comment by rakeshgm** [comment]
we may need to change the services mgmt functions. but these change are okay for now.

File: rgw/v2/utils/utils.py (hunk 1 above)
Comment (mkasturi18) [reply rgw/v2/utils/utils.py:197]: filed BZ  https://bugzilla.redhat.com/show_bug.cgi?id=1942136, to track the issue

**Review by rakeshgm — APPROVED** [review]

**Pull request merged**